assets/capability/capabilities/vacuumcleaner_state.json

```diff
@@ -136,7 +136,7 @@
           "no": "Støvsugeren !{{er|er ikke}}",
           "es": "La aspiradora !{{está|no está}}",
           "da": "Støvsugeren !{{er|er ikke}}",
-          "ru": "Пылесос !{{ |не}}",
+          "ru": "Пылесос !{{|не}}",
```

Comment:
Is this correct?

Reply:
Yes otherwise you'd get duplicate spaces